src/cljs/athens/views/athena.cljs

```diff
@@ -163,19 +163,18 @@
 
 
 (defn create-search-handler
```

Comment:
Please change the name from `create-search-handler` to `search-handler` since it's not creating a handler anymore but just handling.